packages/metal-quartz-link/src/MetalQuartzLink.js

```diff
@@ -0,0 +1,104 @@
+'use strict';
+
+import Component from 'metal-component';
+import MetalQuartzIcon from 'metal-quartz-icon';
+import Soy from 'metal-soy';
+import { core, object } from 'metal';
+import { validators } from 'metal-state';
+
+import templates from './MetalQuartzLink.soy.js';
+
+/**
+ * Implementation of the Metal Quartz Link.
+ */
+class MetalQuartzLink extends Component {
+	/**
+	 * @inheritDoc
+	 */
+	created() {
+		MetalQuartzLink.instances.push(this);
```

Comment:
Hey @pat270, I'm not 100% sure about how good would that be. Depending on how the component re-renders, reference to subcomponents might get lost, outdated or cause inconsistent internal states.

If we do consider this to be implemented at all, I think it should be done in a general way (from within metal.js) with an API based on the `components.ref` information we have, or it can be left to the library user (Liferay Portal), but it should not be a per-component thing.